apps/server/package.json

```diff
@@ -12,6 +12,8 @@
     "express-session": "^1.17.3",
     "express-static-gzip": "^2.1.7",
     "express-validator": "^6.14.2",
+    "google-auth-library": "^9.2.0",
+    "googleapis": "^128.0.0",
```

Comment:
S**t, this adds like 9MB to the app
2.6MB > 11.7MB

Is there anything we can do about this?